src/pages/SelectPage.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import SpotifyWebApi from 'spotify-web-api-js'
+import styled from 'styled-components/macro'
+import Button from '../components/common/Button/Button'
+import Slider from '../components/common/Slider/Slider'
+import Header from '../components/Header/Header'
+
+interface SelectPageProps {
+  spotify: SpotifyWebApi.SpotifyWebApiJs
+  setTracks: React.Dispatch<
+    React.SetStateAction<SpotifyApi.TrackObjectSimplified[] | undefined>
+  >
+}
+
+const SelectPage = ({ spotify, setTracks }: SelectPageProps): JSX.Element => {
+  const [
+    seedObject,
+    setSeedObject,
+  ] = useState<SpotifyApi.RecommendationsOptionsObject>({
+    limit: 20,
+    seed_genres: 'classical',
+  })
+
+  const history = useHistory()
+
+  return (
+    <Container>
+      <Header withBack={false}>Choose your kind</Header>
+      <StyledForm onSubmit={handleSubmit}>
+        <Slider
+          title={'liveness'}
+          startValue={0.1}
+          min={0}
+          max={1}
+          step={0.01}
+          onChange={handleLivenessChange}
+        />
+        <Button>Get your Playlist</Button>
+      </StyledForm>
+    </Container>
+  )
+  function handleLivenessChange(value: number): void {
+    setSeedObject({ ...seedObject, target_liveness: value })
+  }
+
+  function handleSubmit(event: React.FormEvent<{}>): void {
+    event.preventDefault()
```

Comment:
You should do this in the form already.

Reply:
Could you explain this comment? Not sure what is meant by "in the form already" - this function gets called by the form and only the form.

Reply:
I am sorry, I thought that your Form was a function component, but it is just a container. Then everything here is correct. Again: sorry for the confusion.

Reply:
No worries, I get your thought though 👍🏻